### ISSUE TITLE
Countdown timer when taking a picture

### ISSUE THREAD

**Issue by alexstyl** [novoda/peepz#9]
There should be an countdown timer when you click on the :camera: button so that you have time to pose 💪

**Comment by jackSzm** [comment]
Countdown would be nice even when you don't click , so you ALWAYS know when to pose 😄

**Comment by alexstyl** [comment]
Indeed! I don't know if the tab icon is visible in all browsers while you are taking a picture (chrome changes the fav icon to a recording icon). We could use that to display the count down changing to 3...2...1... 📸
